extensions/amp-list/0.1/amp-list.js

```diff
@@ -927,8 +933,9 @@ export class AmpList extends AMP.BaseElement {
     if (this.resizeFailed_) {
       return;
     }
-    const lastItem = dev().assertElement(this.container_.lastChild);
-    this.viewport_.getClientRectAsync(lastItem)
+    const endoOfListMarker = this.container_.lastChild || this.container_;
```

Comment:
If there is no last child (which is possible when the list is empty), use the container instead. It should try to fetch, but will stop correctly if the next url is falsy.